add KILL statement can kill DDL jobs describe (#16124)
pingcap/docs: sql-statements/sql-statement-kill.md

# KILL

> The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to terminate DDL jobs. 

Reviewer: ```suggestion
The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to terminate ongoing DDL jobs. 
```

> The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to terminate ongoing DDL jobs. 

Reviewer: ```suggestion
The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also use the `KILL` statement to terminate ongoing DDL jobs. 
```

The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster.

> **Note:**
>

> > Starting from TiDB v6.2.0, `KILL` statement can kill DDL jobs. 

Reviewer 2: ```suggestion
> Starting from TiDB v6.2.0, `KILL` statement is able to kill DDL jobs. 
```

> The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster.
> 
> > **Note:**
> >
> > Starting from TiDB v6.2.0, `KILL` statement is able to kill DDL jobs. 

Reviewer: ```suggestion
The `KILL` statement is used to terminate a connection in any TiDB instance in the current TiDB cluster. Starting from TiDB v6.2.0, you can also the `KILL` statement to kill DDL jobs. 
```
